src/extension/configuration.c

```diff
@@ -89,6 +89,49 @@ static bool _parse_uint64(
     return _parse_uint(value, decoded_value, LONG_MAX);
 }
 
+static bool _parse_list(
+    zai_string_view value, zval *nonnull decoded_value, bool persistent)
+{
+    zval tmp;
+    ZVAL_ARR(&tmp, pemalloc(sizeof(HashTable), persistent));
+    zend_hash_init(Z_ARRVAL(tmp), 8, NULL,
+        persistent ? ZVAL_INTERNAL_PTR_DTOR : ZVAL_PTR_DTOR, persistent);
+
+    char *data = (char *)value.ptr;
+    if (data && *data) { // non-empty
+        const char *val_start;
+        const char *val_end;
+        do {
+            if (*data != ',' && *data != ' ' && *data != '\t' &&
+                *data != '\n') {
+                val_start = val_end = data;
+                while (*++data && *data != ',') {
+                    if (*data != ' ' && *data != '\t' && *data != '\n') {
+                        val_end = data;
+                    }
+                }
+                size_t val_len = val_end - val_start + 1;
+                zend_string *val_zstr =
+                    zend_string_init(val_start, val_len, persistent);
+                zval val;
+                ZVAL_STR(&val, val_zstr);
```

Comment:
```suggestion
                zval val;
                ZVAL_NEW_STR(&val, zend_string_init(val_start, val_len, persistent));
```

I think you can something along those lines...